ReText/editor.py

```diff
@@ -401,6 +401,23 @@ def updateTextStatistics(self):
 			wordCount += 1
 		self.statistics = (wordCount, alphaNumCount, characterCount)
 
+	def wheelEvent(self, e: QWheelEvent):
```

Comment:
This requires Python 3.5, but the README says that ReText works with 3.2 or higher. Please update the README or remove this.

Reply:
Sorry bout that, I'm used to typed languages :/